cmd/ci-secret-bootstrap/main.go

```diff
@@ -0,0 +1,240 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+
+	coreapi "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	coreclientset "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+
+	"github.com/openshift/ci-tools/pkg/bitwarden"
+	"github.com/openshift/ci-tools/pkg/util"
+)
+
+type options struct {
+	dryRun         bool
+	kubeConfigPath string
+	configPath     string
+	bwUser         string
+	bwPasswordPath string
+
+	bwPassword    string
+	secretClients map[string]secretClient
+	config        config
+}
+
+type secretClient struct {
+	secretsGetter coreclientset.SecretsGetter
+	config        rest.Config
```

Comment:
What is this for? You should be able to just use the client?

Reply:
Really tried. The other one is for test.
Once it changes to getter, I cannot do the `equal` on them because unimported fields.

Reply:
Ignore my non-sense.